### DISCUSSION TITLE
Is POCO project alive again?

### DISCUSSION THREAD

**Discussion by sirzooro** [pocoproject/poco#3736] [General]
I have noticed that last few versions of POCO libraries were released month or less after previous version. Does it mean that POCO libraries have better support now and we can expect active development of them? At the beginning of this year POCO project looked like almost dead, with little activity and issues mostly closed by stale bot after year of inactivity.

**Comment by aleks-f** [comment]
POCO was never dead and will continue to be developed. The level of project activity depends on the availability of core developers and the willingness of the commercial entities using it to sponsor the development.
